mysql-test/suite/galera/t/MDEV-6860.test

```diff
@@ -0,0 +1,42 @@
+--source include/have_innodb.inc
+
+--source include/galera_cluster.inc
```

Comment:
This does not work for MariaDB. As this include will include galera_init.inc that will then include --source include/have_wsrep_enabled.inc that is different than on MySQL version we get [ skipped ]  Test requires wsrep_on=ON. Maybe in MariaDB you need to start 3 node cluster and then take one node out.

Reply:
See galera_as_slave.test there we have:

# As node #1 is not a Galera node, we connect to node #2 in order to run include
/galera_cluster.inc
--connect node_2a, 127.0.0.1, root, , test, $NODE_MYPORT_2
--source include/galera_cluster.inc

Reply:
Actually, it seems test case is correct for 10.4 but not for e.g. 10.2